package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "@babel/preset-env": "^7.13.12",
+    "@babel/preset-react": "^7.14.5",
```

Comment:
We can uninstall this package as we shouldn't need to support JSX in this library

Reply:
Uninstalled